integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataPageSource.java

```diff
@@ -78,10 +78,6 @@ private CarbondataPageSource(List<Type> types, RecordCursor cursor) {
     this.readers = createStreamReaders();
   }
 
-  @Override public long getTotalBytes() {
```

Comment:
upgrade presto version , why need to remove this method ?

Reply:
The Presto 0.186 has remove this method from specification, so we have to remove it from our code as well